<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoDialog</class>
 <widget class="QDialog" name="TodoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add To-Do</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="teamLabel">
     <property name="text">
      <string>                Team:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="teamComboBox">
     <property name="styleSheet">
      <string notr="true">background-color: #2A3240;
  border: 1px solid #3A4A5A;
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED;
    margin-left: 15px;
    margin-right: 15px;

Qcb::drop-down {
    border-left: 1px solid #3A4A5A;
    width: 20px;
}
Qcb:focus {
    border: 1px solid #00FF7F;
    background-color: #323A48;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="todoLabel">
     <property name="text">
      <string>              To-Do Description:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="todoLineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: #2A3240; /* Darker blue-gray for input fields */
    border: 1px solid #3A4A5A; /* Subtle blue-gray border */
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED; /* Light gray text for readability */
    margin-left: 15px;
    margin-right: 15px;
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>             Status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="statusLineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: #2A3240; /* Darker blue-gray for input fields */
    border: 1px solid #3A4A5A; /* Subtle blue-gray border */
    border-radius: 5px;
    padding: 5px;
    color: #E0E6ED; /* Light gray text for readability */
    margin-left: 15px;
    margin-right: 15px;
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="text">
      <string>             Date:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
